MIT.md

```diff
@@ -1,4 +1,4 @@
-## Copyright 2021, [YOUR NAME]
+doit ## Copyright 2021, [YOUR NAME]
 
 ###### Please delete this line and the next one
 ###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
- [ ] @MosDevx great work implementing the changes requested however the MIT license is not customized, kindly remove unnecessary lines and ensure it has the project info eg author-name